packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -126,22 +168,23 @@ export class Table extends Construct {
 
     private readonly keySchema = new Array<dynamodb.TableResource.KeySchemaProperty>();
     private readonly attributeDefinitions = new Array<dynamodb.TableResource.AttributeDefinitionProperty>();
+    private readonly globalSecondaryIndexes = new Array<dynamodb.TableResource.GlobalSecondaryIndexProperty>();
+
+    private readonly nonKeyAttributes: string[] = [];
 
     private readScalingPolicyResource?: applicationautoscaling.ScalingPolicyResource;
     private writeScalingPolicyResource?: applicationautoscaling.ScalingPolicyResource;
 
     constructor(parent: Construct, name: string, props: TableProps = {}) {
         super(parent, name);
 
-        const readCapacityUnits = props.readCapacity || 5;
-        const writeCapacityUnits = props.writeCapacity || 5;
-
         this.table = new dynamodb.TableResource(this, 'Resource', {
             tableName: props.tableName,
             keySchema: this.keySchema,
             attributeDefinitions: this.attributeDefinitions,
+            globalSecondaryIndexes: this.globalSecondaryIndexes,
             pointInTimeRecoverySpecification: props.pitrEnabled ? { pointInTimeRecoveryEnabled: props.pitrEnabled } : undefined,
-            provisionedThroughput: { readCapacityUnits, writeCapacityUnits },
+            provisionedThroughput: { readCapacityUnits: props.readCapacity || 5, writeCapacityUnits: props.writeCapacity || 5 },
```

Comment:
Why is 5 the right default (for example, why not 1 or 10)? Can we refer to justification for this somewhere?

Reply:
I do not know why it was 5 originally, but here is my guess:

- A previous minimum capacity unit was 5 [1], but it is 1 now.
- AWS Free Tier supported up to 5 RCU and WCU before although 25 units are available now [2]. 

If we change it to 1, we could rationalize it from cost perspective. I don't have any strong opinion here.

[1] https://aws.amazon.com/blogs/aws/amazon-dynamodb-reduced-minimum-throughput/
[2] https://aws.amazon.com/free/?utm_source=AWSBlog&utm_medium=socialmedia&SM_AWS_Blog_DDBMin